src/main/scala/ai/privado/model/PrivadoTag.scala

```diff
@@ -108,6 +110,40 @@ object Language extends Enumeration {
   }
 }
 
+object LanguageFileExt extends Enumeration {
+  type LanguageFileExt = Value
+
+  private val JAVA_EXT       = Value(".java")
```

Comment:
File extensions for different languages, useful in generalizing temporary file writing in test.

Reply:
What happens if same frontend can hold different language extensions? Example Javascript frontend can scan both .js and .ts files

Reply:
I have now made this test specific, so every test can specify the language in a separate class called `TestCodeSnippet` with source code and language.